src/api.c

```diff
@@ -121,7 +121,7 @@ yaml_stack_extend(void **start, void **top, void **end)
     void *new_start;
 
     if ((char *)*end - (char *)*start >= INT_MAX / 2)
-	return 0;
+    return 0;
```

Comment:
Isn't the indent broken?

Reply:
Yes, it is. Another reason to fully use 4 spaces / tab / {}, so autoformating tools will not struggle